src/main.zig

```diff
@@ -917,6 +920,10 @@ fn buildOutputType(
                         pagezero_size = std.fmt.parseUnsigned(u64, eatIntPrefix(next_arg, 16), 16) catch |err| {
                             fatal("unable to parse '{s}': {s}", .{ arg, @errorName(err) });
                         };
+                    } else if (mem.eql(u8, arg, "-search_paths_first")) {
+                        search_strategy = .paths_first;
+                    } else if (mem.eql(u8, arg, "-search_dylibs_first")) {
+                        search_strategy = .dylibs_first;
```

Comment:
Perhaps this should be a linker option, not something naked `zig cc` should accept.

Reply:
You are actually pointing to what `build-*` accepts. `zig cc` accepts `-Wl,-search_paths_first` and `-Wl,-search_dylibs_first` and the code for this is here https://github.com/ziglang/zig/blob/08459ff1c21d546c55e2ae954126e121ee88972e/src/main.zig#L1498-L1502